website/static/index.html

```diff
@@ -3,8 +3,28 @@
 
 <head>
     <meta charset="UTF-8">
+    <title>Open VSX Registry | a vendor-neutral open-source alternative to the Visual Studio Marketplace<title>
```

Comment:
The end tag should be `</title>`.

The title is quite long, I'd reduce it to "Open VSX Registry".